lib/assisted_workflow/addons/github.rb

```diff
@@ -22,12 +22,8 @@ def description
       @issue.body.to_s.gsub("\r\n", "\n")
     end
     
-    def other_id
```

Comment:
I removed this `other_id` method. It seems odd to me and a bit confusing. Insted I use [github username](https://github.com/inaka/assisted_workflow/pull/4/files#diff-0843bcf50391700594bd0405b53663f9R48), what do you think?